docs/interactive_engine/tinkerpop/supported_gremlin_steps.md

```diff
@@ -503,8 +503,32 @@ Filter the object in the stream given a biased coin toss.
 Parameters: </br>
 probability - the probability that the object will pass through.
 ```bash
-g.V().coin(0.2) # range is [0, 1]
+g.V().coin(0.2) # range is [0.0, 1.0]
+g.V().out().coin(0.2)
 ```
+
+Usages of the with()-step: </br>
+keyValuePair - the options to set repeatable seed.
+```bash
+g.V().coin(0.2).with('REPEATABLE', 123)
+```
+
+#### [sample()](https://tinkerpop.apache.org/docs/current/reference/#sample-step)
+Generate a certain number of sample results.
+
+Parameters: </br>
+number - allow specified number of objects to pass through the stream.
+```bash
+g.V().sample(10)
+g.V().out().sample(10)
+```
+
+Usages of the with()-step: </br>
+keyValuePair - the options to set repeatable seed.
+```bash
+g.V().sample(10).with('REPEATABLE', 123)
```

Comment:
Add info: what is the supported  keys, and what does each key means (e.g. REPEATABLE), what does it mean

Reply:
Removed this.